### ISSUE TITLE
Switch to Lerna "Fixed/Locked" Release mode

### ISSUE THREAD

**Issue by bd82** [SAP/ui5-language-assistant#411]
## Context

This mono-repo is currently using Lerna's "Independent" mode
- https://github.com/lerna/lerna#independent-mode

This means each sub-package version/life-cycle is managed individually.

While it sounds better in theory, it actually adds too much **complexity**:
- It is difficult to maintain full separation between the packages in PRs thus polluting the CHANGELOG.md files.
- No central aggregated CHANGELOG.
- Required Hack to workaround too many git tag events handling in circle-ci
  - https://github.com/SAP/ui5-language-assistant/blob/master/scripts/trigger-release.js

Independent mode is particularly useful for two cases:
1. When different subsets of sub-packages are consumed by different users as libraries.
   - Which is not the case as the main artifact of this repo is the VSCode extension **not** the sub-package libraries.
     in other words: This is a mono-repo for a VSCode Extension made up of **Multiple modules** (not libraries).
   - It may even be possible to make all the packages in this repo `private`...

2. When a TypeScript **types** package is exported.
    Which does happen in this mono-repo, but due to (1) above it is not targeted at external consumption.

## Open Questions:
- Which version should be used for new releases?
  -  **max**(`map`(`sub-packages`.version)) + 1.0.0 ? (Answer: **3.0.0**)
- Can an aggregated ChangeLog be re-generated from historic SCM data? (Answer: **No**).
- Would the VSCode ext package still contain a CHANGELOG.md? (Answer; **Yes**)
  - If not should the root CHANGELOG.md be copied to it during VSIX bundling?
